src/super_gradients/training/models/pose_estimation_models/dekr_hrnet.py

```diff
@@ -0,0 +1,520 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) Microsoft
+# Licensed under the MIT License.
+# The code is based on HigherHRNet-Human-Pose-Estimation.
+# (https://github.com/HRNet/HigherHRNet-Human-Pose-Estimation)
+# Modified by Zigang Geng (zigang@mail.ustc.edu.cn).
+# ------------------------------------------------------------------------------
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+import torch
+import torch.nn.functional as F
+import torchvision
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.training.models import SgModule
+from torch import nn
+
+__all__ = ["DERKPoseEstimationModel"]
+
+logger = get_logger(__name__)
+
+
+class BasicBlock(nn.Module):
+    expansion = 1
```

Comment:
For BasicBlock, Bottleneck, AdaptBlock and HighResolutionModule:
Would be great to also have a short description + info about input/output if relevant